datadome/resource_custom_rule.go

```diff
@@ -46,11 +113,18 @@ func resourceCustomRule() *schema.Resource {
 		Importer: &schema.ResourceImporter{
 			StateContext: schema.ImportStatePassthroughContext,
 		},
+		Timeouts: &schema.ResourceTimeout{
```

Comment:
just a question:  this is just a best practice timeout, something we checked with dashboard team or other value that is the default by go, but was now previously explicit?

Reply:
The default Timeout values are **20 minutes**.

I can check with dashboard team to specify a more concrete value

Reply:
By agreement with the Dashboard team, it should be set to 60 seconds.